### PR TITLE
Added icons and web app manifest

### LINKED ISSUE #46: Web App Manifest
Create `manifest.json` file and a set of icons (at least 3 sizes). Take this [file](https://github.com/veelenga/bin/blob/master/crystal-ann/logo.jpg) as an original icon.

For more information refer to [The Web App Manifest](https://developers.google.com/web/fundamentals/web-app-manifest/?utm_source=devtools).
For live example refer to [web-application-manifest](https://github.com/GoogleChrome/samples/tree/gh-pages/web-application-manifest).

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #46

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@veelenga tell me what should i use in start url, i will add it 😄

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Crystal Announcements",
+  "short_name": "Crystal ANN",
+  "display": "standalone",
+  "start_url": "",
```
Comment (veelenga) [inline manifest.json]: "start_url":"./",

**Review by veelenga — CHANGES_REQUESTED** [review]
1. Use `./` as a start url.
2. Remove `related_applications`, since we don't have related native application and it produces a warning. See https://github.com/w3c/manifest/issues/591.
2. Move `launcher-icon/` and `manifest.json` to `public/` folder.
2. Use `portrait` orientation instead of `landscape`
3. Tell the browser about the manifest `link rel="manifest" href="/manifest.json"` in [the layout](https://github.com/vaibhavsingh97/crystal-ann/blob/4fed2a9123c8e7d2e8dd0943850c5b7548d63f46/src/views/layouts/application.slang).

Also be free to add yourself to the list of [contributors](https://github.com/crystal-community/crystal-ann#contributors) ;)

Thanks in advanced

**Comment by hugoabonizio** [comment]
Another option is to use ```/?utm_source=homescreen``` as ```start_url``` so we can track this with GA as suggested on https://developers.google.com/web/fundamentals/web-app-manifest/?utm_source=devtools

**Comment by vaibhavsingh97** [comment]
@veelenga Please review

**Review by veelenga — APPROVED** [review]

**Comment by veelenga** [comment]
Now this works perfectly in Chrome:

screen shot 2017-10-03 at 6 31 23 pm

Good job. Thanks for contribution!

**Pull request merged**